src/fragments/lib/logging/ios/flushlogs/flushlogs.mdx

```diff
@@ -0,0 +1,13 @@
+If there are use cases where logs need to be sent immediately, you can use the `AWSCloudWatchLoggingPlugin` instance to force an automatic log flush.
```

Comment:
Let us start with an explanation first. 

When using the Amplify Logger, all logged messages are saved locally on the user's device first and then flushed at a set interval that you can customize (then link to that section). You can also choose to flush events manually if needed by following the steps outlined in this section.

Reply:
Added intro paragraph.